res/values/strings.xml

```diff
@@ -8,25 +8,31 @@
   <string name="pref_category_about">About</string>
   <string name="pref_category_other">Other Options</string>
   <string name="pref_category_time">Game Time</string>
-  <string name="pref_dialog_title_delay_length">Enter Time (seconds)</string>
+  <string name="pref_dialog_title_enter_time">Enter Time</string>
   <string name="pref_dialog_title_delay_type">Select Time Delay</string>
-  <string name="pref_dialog_title_starting_time">Enter Time (minutes)</string>
+  <string name="pref_dialog_title_time_units">Select Time Units</string>
   <string name="pref_summary_alert_ringtone">Set the ringtone to be played when time expires.</string>
   <string name="pref_summary_delay_length">Set the amount of time used for time delays.</string>
   <string name="pref_summary_delay_type">Set the time delay scheme to use during play.</string>
+  <string name="pref_summary_delay_length_units">Set units in which to specify delay length.</string>
   <string name="pref_summary_haptic_feedback">Vibrate slightly on button presses.</string>
   <string name="pref_summary_black_background">Saves power on OLED screens.</string>
+  <string name="pref_summary_show_deciseconds">Display tenths of seconds when clock/delay below 10 seconds.</string>
```

Comment:
Should this be "[...] when clock/delay is below [...]"? English is not my first language, so I'm not sure.

Reply:
Both are acceptable, but "... is ..." is more proper. Changed it.